python/GafferUI/PlugValueWidget.py

```diff
@@ -676,6 +656,45 @@ def __drop( self, widget, event ) :
 
 		return True
 
+	# Type registry internals
+
+	@classmethod
+	def __creator( cls, plug, useTypeOnly ) :
+
+		# First try to create one using a creator registered for the specific plug.
+		if not useTypeOnly :
+
+			widgetType = Gaffer.Metadata.value( plug, "plugValueWidget:type" )
+			if widgetType is None :
+				widgetType = Gaffer.Metadata.value( plug, "layout:widgetType" )
+				if widgetType is not None :
+					warnings.warn( "The \"layout:widgetType\" metadata entry is deprecated, use \"plugValueWidget:type\" instead.", DeprecationWarning )
```

Comment:
Time to remove, as its been deprecated a while?

Reply:
I'm frying other :fish: here. But yep, if you'd like to make a PR, please do :)